packages/xlayers/src/app/editor/editor.component.ts

```diff
@@ -170,15 +174,23 @@ export class EditorComponent implements OnInit {
   }
 
   ZoomIn() {
+    this.zoomIn = !this.zoomIn;
+    setTimeout(() => this.zoomIn = !this.zoomIn, this.DEFAULT_TIMEOUT);
```

Comment:
Can we have a visual indication when the zoom in is less than 1 (reset value) and when the zoom out is higher than 1 (reset value)?

Reply:
Instead of this behavior? Of course we can do this when we just look at the state of the application.

Reply:
I was thinking about this when I saw my this in Chrome 
![image](https://user-images.githubusercontent.com/1699357/53970946-1af96900-40fc-11e9-8e2e-813da9a3a2c7.png)

Reply:
![image](https://user-images.githubusercontent.com/17835373/53971074-67dd3f80-40fc-11e9-83f4-3e0f921ca7da.png)


I clicked twice on zoom. So zoomlevel is higher than 1. It stays in red until you reset it or it drops below 1.